main.py

```diff
@@ -86,7 +91,14 @@
         ConvertPacBioBamToCram,
     ],
     'validation': [ValidationMtToVcf, ValidationHappyOnVcf],
-    'large_cohort': [LoadVqsr, Frequencies, AncestryPlots, GvcfMultiQC, CramMultiQC],
+    'large_cohort': [
+        LoadVqsr,
+        Frequencies,
+        AncestryPlots,
+        GvcfMultiQC,
+        CramMultiQC,
+        PrepareBrowserTable,
+    ],
```

Comment:
As noted in the comment above, I would drop `PrepareBrowserTable` from here, and create a separate pipeline:

```
'large_cohort': [
        LoadVqsr,
        Frequencies,
        AncestryPlots,
        GvcfMultiQC,
        CramMultiQC,
    ],
    ...
    ...
 'prepare_browser_table': [
        PrepareBrowserTable,
    ],   
```